src/main/resources/google_checks.xml

```diff
@@ -47,6 +50,19 @@
   </module>
 
   <module name="TreeWalker">
+    <module name="SuppressWarningsHolder" />
+    <module name="SuppressionCommentFilter">
+      <property name="offCommentFormat" value="CHECKSTYLE.OFF\: ([\w\|]+)" />
+      <property name="onCommentFormat" value="CHECKSTYLE.ON\: ([\w\|]+)" />
+      <property name="checkFormat" value="$1" />
+    </module>
+    <!-- Allow suppression using a comment on the same line or the line before -->
+    <module name="SuppressWithNearbyCommentFilter">
+      <property name="commentFormat" value="CHECKSTYLE.OFF\: ([\w\|]+)"/>
+      <property name="checkFormat" value="$1"/>
+      <property name="influenceFormat" value="1"/>
```

Comment:
Please explain why we do not leave properties as default value (except for comment format)

Reply:
done